src/darwin/Framework/CHIP/MTRBaseDevice.h

```diff
@@ -234,8 +234,12 @@ typedef NS_ENUM(uint8_t, MTRTransportType) {
  *
  * @param completion  response handler will receive either values or error.
  *
- *                    Received values are documented in the definition of
- *                    MTRDeviceResponseHandler.
+ *                    A path-specific error status will get turned into an error
+ *                    passed to the completion, so values will only be passed in
```

Comment:
So the completion handler still gets an array rather than a single path/value pair?

Reply:
There is no "value"... The completion handler gets an array containing a single dictionery, which has the path in it (as a "path key": path pair).